plugins/@grouparoo/pipedrive/src/lib/export/organizations/destinationMappingOptions.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseType,
+  SimpleAppOptions,
+  objectCache,
+} from "@grouparoo/core";
+
+import { connect } from "../../connect";
+import { PipedriveClient } from "../../client";
+import { GROUP_FIELD_PREFIX } from "./listMethods";
+
+export interface PipedriveCacheData {
+  appId: string;
+  appOptions: SimpleAppOptions;
+}
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod =
+  async ({ appOptions, appId }) => {
+    const client = await connect(appOptions);
+    const allKnown = await getKnownOrganizationFields(
+      client,
+      { appId, appOptions },
+      true
+    );
+
+    const required = allKnown.filter((field) => field.required);
```

Comment:
Think there's some code to share here with `Persons` ? Alot of the code is probably similiar. Getting all the fileds for example and at the bottom as well.